### ISSUE TITLE
error: initialization of ‘void (*)(struct tty_struct *, const struct ktermios *)’

### ISSUE THREAD

**Issue by NCJECulver** [freemed/tty0tty#50]
Kernel: 6.2.0-25-generic x86_64
gcc: (Ubuntu 12.3.0-1ubuntu1~23.04) 12.3.0

After cloning, building module fails:
```
git clone https://github.com/freemed/tty0tty.git
cd tty0tty/module/
make
. . . 
tty0tty.c:608:24: error: initialization of ‘void (*)(struct tty_struct *, const struct ktermios *)’ from incompatible pointer type ‘void (*)(struct tty_struct *, struct ktermios *)’ [-Werror=incompatible-pointer-types]
  608 |         .set_termios = tty0tty_set_termios,

```
Changing line 291 from:

```
struct ktermios *old_termios)
```

to

```
const struct ktermios *old_termios)
```

resolved the error.

![error](https://github.com/freemed/tty0tty/assets/61895881/82947e4c-7a38-4a90-8f5a-ced87e70b293)
![errorfixed](https://github.com/freemed/tty0tty/assets/61895881/76c88e52-148b-4f64-89d1-c65cbd27c7b7)

**Comment by colinleroy** [comment]
This is addressed in #49

**Comment by stappersg** [comment]
Fixed in #57
